docs/content/any/guides/data_filtering/_index.md

```diff
@@ -0,0 +1,248 @@
+---
+title: "Data Filtering Preview"
+weight: 1
+showContentForAnyLanguage: true
+---
+
+# Data Filtering Preview
+
+{{< ifLangExists >}}
+{{% ifLang not="node" %}}
+{{% ifLang not="python" %}}
+{{% ifLang not="ruby" %}}
+
+Data filtering is coming soon for {{< lang >}}!
+
+If you want to get data filtering in your app now or just want to
+register your interest for Data Filtering in {{< lang >}} [drop into our Slack](http://join-slack.osohq.com) or
+<a href="mailto:engineering@osohq.com?subject=Data%20filtering%20support%20for%20{{< currentLanguage >}}&body=I%27m%20interested%20in%20data%20filtering%20support%20for%20{{< currentLanguage >}}">send an email</a>.
+to our engineering team and we'll unblock you.
+{{% /ifLang %}}
+{{% /ifLang %}}
+{{% /ifLang %}}
+{{% /ifLangExists %}}
+
+{{% callout "Early Preview" "blue" %}}
+
+Data filtering is currently an Early Preview. If you have any trouble using it or it doesn't work with your policy [drop into our Slack](http://join-slack.osohq.com) or
```

Comment:
Instead of this paragraph, let's add this callout to the top of the page. That'll make it consistent with the warning about enforcement.

```
{{% callout "Note: 0.20.0 Beta Feature" %}}
  This is an API provided by the beta release of Oso 0.20.0, meaning that it is
  not yet officially released. You may find other docs that conflict with the
  guidance here, so proceed at your own risk! If you have any questions, don't
  hesitate to [reach out to us on Slack](https://join-slack.osohq.com). We're
  here to help.
{{% /callout %}}

<div class="pb-10"></div>
```